central/role/resources/list.go

```diff
@@ -23,21 +23,37 @@ import (
 //
 // KEEP THE FOLLOWING LIST SORTED IN LEXICOGRAPHIC ORDER (case-sensitive).
 var (
-	// Access is the new resource grouping all access related resources.
+	// Access groups all access-related resources. It aims to cover
+	// configuration for authentication and authorization. For instance,
+	// it has replaced: AuthProvider, Group, Licenses, Role, User.
 	Access = newResourceMetadata("Access", permissions.GlobalScope)
-	// Administration is the new resource grouping all administration-like resources.
+
+	// Administration groups all administration-like resources except those
+	// related to authentication and authorization. It aims to cover platform
+	// configuration. For instance, it has replaced: AllComments, Config,
+	// ComplianceRunSchedule, DebugLogs, NetworkGraphConfig, ProbeUpload,
+	// ScannerBundle, ScannerDefinitions, SensorUpgradeConfig, ServiceIdentity.
 	Administration = newResourceMetadata("Administration", permissions.GlobalScope)
```

Comment:
Have we considered two separate administration resources—one for workflow management and one for ACS setup?
The persona for workflow management and ACS root-level configurations are different.

Reply:
We have not until now but this sounds like a very reasonable idea.